client/db_test.go

```diff
@@ -0,0 +1,28 @@
+package client_test
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/client"
+	"github.com/cockroachdb/cockroach/server"
+)
+
+func ExampleDB() {
+	s := server.StartTestServer(nil)
+	defer s.Stop()
+
+	db := client.Open(s.ServingAddr())
+	db.Put("hello", "world")
```

Comment:
This is a little confusing...using "world" as a value and a key. I'd rejigger this slightly.